reproductions/basic-sqlite/yarn.lock

```diff
@@ -0,0 +1,139 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Stray `yarn.lock` file in a pnpm workspace. If you need to add a test that specifically uses yarn for any reason, you could add support for yarn packages in `packages/client/tests/e2e` and create a test there, but I don't believe that should be necessary. Reading the issue, a regular test in `packages/cli/src/__tests__/commands/Version.test.ts` should be sufficient.